Add payment type and wire up transaction cell

jewsroch opened the pull request:
Added payment type model and pointer from transaction model. (also added in parse)
Wired up TransactionCell to view data and tableView.

jewsroch committed:
Add payment type and wire up transaction cell

Added payment type model and pointer from transaction model. (also added in parse)
Wired up TransactionCell to view data and tableView.

jewsroch commented on PerDiem/PerDiem/Controllers/TransactionsViewController.m:
```diff
@@ -26,6 +29,10 @@ - (void)viewDidLoad {
     self.edgesForExtendedLayout = UIRectEdgeNone;
     [self setupNavigationBar];
+
+    // @Todo Fetch Transactions
```
Need to finish this, but wasn't sure how we were planning to organize the models for this common fetch.

frojasg replied:
after merging #13 you can use 

``` objective-c
        [Transaction transaction:^(NSArray *transactions, NSError *error) {
            for (Transaction *transaction in transactions) {
                NSLog(@"%@", budget);
            }
        }];
```

jewsroch commented on PerDiem/PerDiem/TransactionCell.m:
```diff
@@ -14,6 +25,27 @@ - (void)awakeFromNib {
+    self.paymentTypeLabel.text = self.transaction.paymentType.name;
+    self.summaryLabel.text = self.transaction.summary;
+
+    // @TODO - make common currency formatter method to be used throughout app.
```
We should make some common helpers for currency and date formatting. 
Also, we should figure out the best way to easily make all currency strings localized.

flochtililoch replied:
here's an example on how to create a currency formatter:

in private interface:

``` objective-c
@property (strong, nonatomic) NSNumberFormatter *amountFormatter;
```

in implementation:

``` objective-c
- (NSNumberFormatter *)amountFormatter {
    if (!_amountFormatter) {
        _amountFormatter = [[NSNumberFormatter alloc] init];
        [_amountFormatter setNumberStyle: NSNumberFormatterCurrencyStyle];
    }
    return _amountFormatter;
}
```

flochtililoch commented:
:shipit:

jewsroch committed:
Merge branch 'master' into payment-type-and-transactions

Conflicts:
	PerDiem/PerDiem/models/Transaction.h

jewsroch committed:
Add Transaction fetch in transactions list view

Also fixed minor bug on transaction entry view. I only had one budget so it wasn't allowing me to submit transactions.
